src/framework/application.js

```diff
@@ -1565,29 +1577,6 @@ Object.assign(pc, function () {
             }
         },
 
-        /**
-         * @function
-         * @name pc.Application#enableVr
-         * @description Create and assign a {@link pc.VrManager} object to allow this application render in VR.
-         */
-        enableVr: function () {
-            if (!this.vr) {
-                this.vr = new pc.VrManager(this);
-            }
-        },
-
-        /**
-         * @function
-         * @name pc.Application#disableVr
-         * @description Destroy the {@link pc.VrManager}.
-         */
-        disableVr: function () {
-            if (this.vr) {
-                this.vr.destroy();
-                this.vr = null;
-            }
-        },
-
```

Comment:
For any API that you remove, can you add stub functions that print a debug warning to src\backwards-compatibility.js please?

Reply:
Added some warnings.